cachi2/core/models/output.py

```diff
@@ -157,7 +147,24 @@ def _unique_project_files(cls, project_files: list[ProjectFile]) -> list[Project
         """Sort and de-duplicate project files by path."""
         return unique_sorted(project_files, by=lambda f: f.abspath)
 
+
+class RequestOutput(pydantic.BaseModel):
+    """Results of processing one or more package managers."""
+
+    build_config: BuildConfig
+    sbom: Sbom
+
     @classmethod
     def empty(cls):
         """Return an empty RequestOutput."""
-        return cls(packages=[], environment_variables=[], project_files=[])
+        return cls(sbom=Sbom(), build_config=BuildConfig())
+
+    @classmethod
+    def create(cls, components=[], environment_variables=[], project_files=[]) -> "RequestOutput":
```

Comment:
This is missing type hints for the params

And mutable defaults (the lists) are almost always a bad idea